specification/deviceprovisioningservices/data-plane/Microsoft.Devices/preview/2021-11-01-preview/device.json

```diff
@@ -0,0 +1,1044 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2021-11-01-preview",
+    "title": "ProvisioningDeviceClient",
+    "description": "API for device runtime operations with the Azure IoT Hub Device Provisioning Service"
+  },
+  "host": "global.azure-devices-provisioning.net",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/{idScope}/devices/{registrationId}/certificates/{certificateId}": {
+      "get": {
+        "tags": [
+          "GET"
+        ],
+        "summary": "Obtains the issued certificate.",
+        "operationId": "CertificateIssuance_Get",
+        "consumes": [],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "name": "idScope",
+            "in": "path",
+            "description": "The service assigned ID scope.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "registrationId",
+            "in": "path",
+            "description": "The device registration ID. A case-insensitive string (up to 128 characters long) of alphanumeric characters plus certain special characters : . _ -. No special characters allowed at start or end.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "certificateId",
+            "in": "path",
+            "description": "Certificate ID. A case-insensitive string (up to 128 characters long) of alphanumeric characters plus certain special characters : . _ -. No special characters allowed at start or end.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success",
+            "schema": {
+              "$ref": "#/definitions/CertificateIssuanceResponse"
+            }
+          },
+          "default": {
+            "description": "Error response",
+            "schema": {
+              "$ref": "#/definitions/ProvisioningServiceErrorDetails"
+            },
+            "headers": {
+              "x-ms-error-code": {
+                "description": "The error code for specific error that occurred.",
+                "type": "string"
+              }
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Returns the issued certificate": {
+            "$ref": "./examples/device/CertificateIssuance/getIssuedCertificate.json"
+          }
+        }
+      }
+    },
+    "/{idScope}/devices/{registrationId}/operations/{operationId}": {
+      "get": {
+        "tags": [
+          "GET"
+        ],
+        "summary": "Obtains the device operation status.",
+        "operationId": "Device_OperationStatusLookup",
```

Comment:
Seems to follow a different pattern than the previous operation ID. Normally, these follow the form of "<Topic>_<Operation e.g., 'Get' or 'List'>" which also helps form the SDK method names.